src/browser/client.ts

```diff
@@ -49,13 +48,13 @@ function handleCallbackResponse() {
 function checkCallbackResponse() {
   let params;
   if (window.location.hash) {
-    params = qs.parse(window.location.hash.substring(1));
-    if (params.access_token) {
+    params = new URLSearchParams(window.location.hash.substring(1));
+    if (params.get('access_token')) {
       return { success: true, body: params };
     }
   } else if (window.location.search) {
-    params = qs.parse(window.location.search.substring(1));
-    if (params.error) {
+    params = new URLSearchParams(window.location.search.substring(1));
+    if (params.get('error')) {
```

Comment:
the changes in this file are to swap `querystring` (deprecated) with `URLSearchParams` which is available on browsers and node:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

no breaking changes